R/derive_vars_aage.R

```diff
@@ -211,11 +211,10 @@ derive_var_age_years <- function(dataset, age_var, age_unit = NULL, new_var) {
 
   if (!unit_var %in% colnames(dataset)) {
     if (is.null(age_unit)) {
-      err_msg <- paste(
-        "There is no variable unit:", unit_var, "associated with", age_var,
-        "and the argument `age_unit` is missing. Please specify a value for `age_unit`"
-      )
-      abort(err_msg)
+      cli_abort(paste(
+        "There is no unit variable ({.var {unit_var}}) associated with {.var {age_var}}",
+        "and the {.arg age_unit} argument is missing. Please specify a value for {.arg age_unit}."
+      ))
```

Comment:
```suggestion
      cli_abort(paste(
         "There is no unit variable ({.var {unit_var}}) associated with {.var {age_var}}".
         "Please specify a value for {.arg age_unit}."
      ))
```

Reply:
Maybe?  I'm open to keep - just found this message a bit long for missing something